### ISSUE TITLE
Failed To Open

### ISSUE THREAD

**Issue by SirPlancake** [WilliamVenner/gmpublisher#129]
It fails to open on Windows 11.

**Comment by WilliamVenner** [comment]
I'm on Windows 11 and it works for me. I can't do anything with the information that you have given.

**Comment by SirPlancake** [comment]
When I try to open it the window just disappears. I have literally everything installed.

**Comment by WilliamVenner** [comment]
Install this: https://go.microsoft.com/fwlink/p/?LinkId=2124703

**Comment by WilliamVenner** [comment]
Check `%temp%/gmpublisher/logs`

**Comment by SirPlancake** [comment]
!!!!!!!!!!!!! PANIC !!!!!!!!!!!!!
panicked at 'called `Result::unwrap()` on an `Err` value: CreateWebview(WebView2Error(webview2 error, HRESULT 0x80070002))', src\main.rs:130:10
   0:     0x7ff71be077cf - <unknown>
   1:     0x7ff71be07702 - <unknown>
   2:     0x7ff71c1426d2 - <unknown>
   3:     0x7ff71c40427e - <unknown>
   4:     0x7ff71c404005 - <unknown>
   5:     0x7ff71c40150f - <unknown>
   6:     0x7ff71c403cb4 - <unknown>
   7:     0x7ff71c55ee75 - <unknown>
   8:     0x7ff71c55eff3 - <unknown>
   9:     0x7ff71c14078b - <unknown>
  10:     0x7ff71bf21ae6 - <unknown>
  11:     0x7ff71bf24b2c - <unknown>
  12:     0x7ff71c3f1ee0 - <unknown>
  13:     0x7ff71c159657 - <unknown>
  14:     0x7ff71c55c60c - <unknown>
  15:     0x7ffa22be54e0 - BaseThreadInitThunk
  16:     0x7ffa23ee485b - RtlUserThreadStart

**Comment by WilliamVenner** [comment]
Yeah, 0x80070002 means WebView2 isn't installed, so I'm not sure what's wrong with your installation.

**Comment by SirPlancake** [comment]
Yeah, I don't know what its doing because it says web view 2 is already installed.

**Comment by SirPlancake** [comment]
Should I reinstall web view 2?
